common/src/main/res/values/colors.xml

```diff
@@ -9,6 +9,7 @@
 
     <!-- System -->
     <color name="android_system_accent">#1F78FF</color>
+    <color name="default_system_blue_dark">#0A84FF</color>
```

Comment:
I suggest to avoid direct using names of colors such as "blue", "red" etc. since in light theme it could be another color

Reply:
Thats the name from Figma, I will notify Andrey about that